gitpod-io/gitpod
# [dashboard] Onboarding: new video onboarding for new workspace

## Description


This PR adds a new video onboarding feature for new workspaces. The feature includes a YouTube video that provides an overview of Gitpod and how to create a new workspace. When the video is played, an event is tracked using the `trackVideoClick` function.

When someone clicks on YT video, it would trigger a segment event with the following payload as event `video_clicked`:

```json
{
  "anonymousId": "895ajdj47-14yy-4bab-b113-xxxxxx",
  "event": "video_clicked",
  "properties": {
    "context": "create-new-workspace",
    "path": "/workspaces"
  }
}
```

**ACTION REQUIRED:** Update YouTube Video

## Related Issue(s)

Fixes OPM-655

## How to test


- Open preview environment.
- Complete initial steps
- After completion, you will see a video with updated copy text on `/workspaces`

## Documentation


#### Preview status

<p>Gitpod was successfully deployed to your preview environment.</p>
<ul>
	<li><b>🏷️ Name</b> - siddhant-o6097563f27</li>
	<li><b>🔗 URL</b> - <a href="https://siddhant-o6097563f27.preview.gitpod-dev.com/workspaces" target="_blank">siddhant-o6097563f27.preview.gitpod-dev.com/workspaces</a>.</li>
	<li><b>📚 Documentation</b> - See our <a href="https://www.notion.so/gitpod/6debd359591b43688b52f76329d04010#7c1ce80ab31a41e29eff2735e38eec39" target="_blank">internal documentation</a> for information on how to interact with your preview environment.</li>
	<li><b>📦 Version</b> - siddhant-opm-655-add-video-to-dashboard-when-onboarding-gha.26994</li>
	<li><b>🗒️ Logs</b> - <a href="https://console.cloud.google.com/logs/query;query=jsonPayload.kubernetes.host%3D%22preview-siddhant-o6097563f27%22%0A%0A--%20Filter%20on%20service:%0A--%20jsonPayload.serviceContext.service%3D%22ws-manager-mk2%22%0A;duration=P1D?project=gitpod-dev-preview" target="_blank">GCP Logs Explorer</a></li>
</ul>

## Build Options

<details>
<summary>Build</summary>

      Run the build with werft instead of GHA
      Run Leeway with `--dont-test`
</details>

<details>
<summary>Publish</summary>

</details>

<details>
<summary>Installer</summary>

  Add desired feature flags to the end of the line above, space separated
</details>

<details>
<summary>Preview Environment / Integration Tests</summary>

      If enabled this will build `install/preview`
      If enabled this will create the environment on GCE infra
      Saves cost. Untick this only if you're really sure you need a non-preemtible machine.
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`. If enabled, `with-preview` and `with-large-vm` will be enabled.
</details>

/hold
